release.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# If you haven't already read DEVELOPMENT.md, please stop and do so. Running
+# this script is _not_ the first thing you need to do to make a release.
```

Comment:
Can you make this a prompt instead? "Have you read DEVELOPMENT.md first? [y/N]"

Reply:
Done.

Reply:
haha, sorry, that's not exactly what I meant - I mean a Bash prompt for yes/no in case people run this script without reading the comments you have here.

e.g. https://github.com/sourcegraph/syntect_server/blob/master/publish.sh#L42-L49

Reply:
That makes more sense! Updated. :smile: